app/controllers/collaborators_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
   def create
     @collaborator = Collaborator.new(collaborator_params)
     @collaborator.user = current_user
-    if @collaborator.save and current_user.profile.nil?
```

Comment:
Hay que cuidar el utilizar "and" en el código. Si hay una diferencia semántica entre el "&&" y el "and". El "and" tiene menor precedencia que "&&' incluso menor que "=". Es recomendado el evitar el uso de este operador booleano. http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby